benchmarks/benchmarks/api/bench_edge_ids.py

```diff
@@ -0,0 +1,33 @@
+import time
+import dgl
+import torch
+import numpy as np
+
+from .. import utils
+
+# edge_ids is not supported on cuda
+@utils.skip_if_gpu()
+@utils.benchmark('time', timeout=1200)
+@utils.parametrize_cpu('graph_name', ['cora', 'livejournal', 'friendster'])
+@utils.parametrize_gpu('graph_name', ['cora', 'livejournal'])
+@utils.parametrize('format', ['csc'])  # csc is not supported
+@utils.parametrize('fraction', [0.01, 0.1])
+def track_time(graph_name, format, fraction):
+    device = utils.get_bench_device()
+    graph = utils.get_graph(graph_name, format)
+    graph = graph.to(device)
+    eids = np.random.RandomState(6666).choice(
+        np.arange(graph.num_edges(), dtype=np.int64), int(graph.num_edges()*fraction))
```

Comment:
We have set the seed at here: https://github.com/dmlc/dgl/blob/master/benchmarks/benchmarks/utils.py#L251

Reply:
Removed RandomState